_includes/tags.html

```diff
@@ -19,11 +19,19 @@
                        using the literal string '/blog/'. Path is used to
                        create the filtered URL: {{ path }}?filter_tags={{ tag }}
 
+   display_header:     The optional boolean flag to display the tags_header element. 
+                       Defaults to True. Adds class 'tags__hide-header' if falsy 
+                       value passed (e.g., '', false, None)
```

Comment:
If you passed false in in python is it `false` or `False` or does it not matter?